templates/threejs/packages/client/src/App.tsx

```diff
@@ -5,23 +5,106 @@ import { getComponentValueStrict, Has } from "@latticexyz/recs";
 import { useMUD } from "./MUDContext";
 import { useKeyboardMovement } from "./useKeyboardMovement";
 
-const Plane = (props: ThreeElements["mesh"]) => {
+const Plane = () => {
   return (
-    <mesh {...props}>
+    <>
       {/* eslint-disable-next-line react/no-unknown-property */}
-      <boxGeometry args={[10, 5, 10]} />
-      <meshStandardMaterial color="green" />
-    </mesh>
+      <mesh position={[0, -1, 0]}>
+        {/* eslint-disable-next-line react/no-unknown-property */}
```

Comment:
can we figure out how to get react+eslint to allow this? I assume there's a three.js eslint config to include here

Reply:
Done.

Reply:
And the solution I found online doesn't work. Can we *please* just not worry about it? Prettier here is more trouble than it is worth, IMAO.

Reply:
I've always run into this issue with ThreeJS and TypeScript, I'll play around and try fix it. Also I get errors for all the elements so we are going to need a lot of these comments otherwise!
<img width="564" alt="image" src="https://github.com/latticexyz/mud/assets/29184158/5976aca2-7337-4404-983e-b521578e8b47">

Reply:
Sounds like an open issue and the [@react-three/eslint-plugin](https://socket.dev/npm/package/@react-three/eslint-plugin) didn't work for me: https://github.com/pmndrs/react-three-fiber/discussions/2487

For now I added a comment at the top of the file but we could also add a `eslintrc` rule like 
```
  ts"react/no-unknown-property": ["off", { "ignore": ["JSX"] }],
```